api/api/admin/__init__.py

```diff
@@ -134,3 +145,32 @@ class AccessTokenAdmin(admin.ModelAdmin):
         "created",
         "updated",
     )
+
+
+@admin.register(UserPreferences)
+class IndividualUserPreferencesAdmin(admin.ModelAdmin):
+    """
+    Model admin for showing user preferences. This should only ever show the
+    currently logged-in user's preferences
+    """
+
+    can_delete = False
+    verbose_name_plural = "My Preferences"
+    verbose_name = "My Preferences"
+    form = UserPreferencesAdminForm
+
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.filter(user=request.user)
+
+    def has_change_permission(self, request, obj=None):
+        if not obj:
+            # the changelist itself
+            return True
+        return obj.user == request.user
+
+    def changelist_view(self, request, extra_context=None):
+        obj = self.get_queryset(request).first()
```

Comment:
I think this will cause an issue in live environments with the existing users, because they won't have had the `create` receiver triggered that will create a preferences instance for _new_ users. We could solve this with a data migration that creates a user preferences instance for all existing users, and add it to the `0058` migration.